src/lib/components/Login/OnaOauthLogin/OnaOauthLogin.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { useOAuthLogin } from '@onaio/gatekeeper/dist/components/login';
-import { AuthorizationGrantType } from '@onaio/gatekeeper';
+import { useOAuthLogin, AuthorizationGrantType } from '@onaio/gatekeeper/dist/components/login';
```

Comment:
I think these need to come from `@onaio/gatekeeper`, not a `dist` folder. Please make sure that gatekeeper is exporting `AuthorizationGrantType` from its `index`, bump the gatekeeper package version and publish it, reinstall here, and pull both imports directly from `@onaio/gatekeeper`. 🙏

Reply:
@cKellyDesign I can see they are both exported. I can also see the newest version is `0.0.19`. I am going to upgrade.

Reply:
Sorry..`useOAuthLogin` is missing